_pkgdown.yml

```diff
@@ -1,39 +1,71 @@
-url: https://pharmaverse.github.io/admiraltemplate
+url: https://pharmaverse.github.io/admiraloptha
 
 template:
   bootstrap: 5
   params:
     bootswatch: flatly
+search:
+  exclude: ['news/index.html']
 repo:
   url:
-    home: https://github.com/pharmaverse/admiraltemplate
-    source: https://github.com/pharmaverse/admiraltemplate/blob/main/
-    issue: https://github.com/pharmaverse/admiraltemplate/issues/
+    home: https://github.com/pharmaverse/admiralonco
+    source: https://github.com/pharmaverse/admiralonco/blob/test/
+    issue: https://github.com/pharmaverse/admiralonco/issues/
```

Comment:
It should be `admiralophta` instead of `admiralonco` and `main` instead of `test` for `source:`.